.env

```diff
@@ -7,4 +7,7 @@ REACT_APP_API_HOSTNAME="https://localhost:8087"
 # REACT_APP_ALLOW_DEV_TOOLS=false
 
 # Google Analytics' Universal ID
-ANALYTICS_ID="UA-208192118-1"
\ No newline at end of file
+ANALYTICS_ID="UA-208192118-1"
+
+# GeoLocation Rapid Api keu
+GEO_API_KEY="APIKEY"
```

Comment:
This will not work unless you manually add it to the process.env object (@BrunoRosendo did it for the analytics one) 

If you want the easy way, you need to prefix this with REACT_APP (and change the usage as well)

Reply:
If I have the api key in .env.local it works

Reply:
I added it into the process env actually

Reply:
From what I can see you only added it to our config object (assuming it was already in process.env)

Check if it works in the deploy preview as well, if it does, maybe they updated it, because it didn't in the past (you can see the prefix in other env vars)

Reply:
The deploy preview won't have any valid key in its .env, right?